### PR TITLE
Render missions

### PR DESCRIPTION
**Pull request by stephenkati**
- Used **`useSelector()` Redux Hook** to select the state slices and render lists of missions.
- Used **CSS** to style missions table.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -25,3 +25,58 @@ code {
...
+.mission-list td {
+  padding: 0.5em 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 18px;
+  min-width: 100px;
+}
+
+.mission-list:nth-child(even) {
+  background-color: rgb(168, 168, 168);
+}
```
Comment (rugwizangoga) [inline src/index.css]: The background color used here does not correspond to the color suggested in the template which looks more like this: #f2f2f2

File: src/index.css [hunk 2]
```diff
@@ -25,3 +25,58 @@ code {
...
+td {
+  border: 1px solid rgb(131, 131, 131);
+}
+
+.mission-table thead tr {
+  text-align: left;
+}
+
+.table-head {
+  padding: 0.5em 10px;
```
Comment (rugwizangoga) [inline src/index.css]: I would suggest to move all these into the app.css and delete this index.css in order to have all your stylings into one file which help in code maintenance and reusability.

File: src/index.css (hunk 1 above)
Comment (stephenkati) [reply src/index.css]: thanks for pointing that out, I'm on it

File: src/index.css (hunk 2 above)
Comment (stephenkati) [reply src/index.css]: good idea

**Review by rugwizangoga — CHANGES_REQUESTED** [review]
## congratulation your codes looks clean 
alonzo-lerone-this-close


- Have used the right GitHub flow ✅
- Meaningful summary for PR ✅
- Professional README.md file ✅
- Linter checks passed ✅
- Gitflow followed ✅

## Required changes
- you will find the required in the comments

**Review by rugwizangoga — APPROVED** [review]
## congratulation your codes looks clean 
great job

- Have used the right GitHub flow ✅
- Meaningful summary for PR ✅
- Professional README.md file ✅
- Linter checks passed ✅
- Gitflow followed ✅

**Pull request merged**